metadata/multirepo/multirepo.go

```
@@ -311,8 +311,14 @@ func (client *MultiRepoClient) DownloadTarget(repos []string, targetFile *metada
 	log := metadata.GetLogger()
 
 	for _, repoName := range repos {
+		repoClient, ok := client.TUFClients[repoName]
+		if !ok {
+			log.Error(fmt.Errorf("client not found"), "No client found for this repo", "repo", repoName)
+			continue
```

Don't we want to return an error here instead of continuing?

---

Thank you! 💯